charts/gsp-cluster/datasources/cloudwatch.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: 1
+
+datasources:
+  - name: Cloudwatch
+    type: cloudwatch
+    jsonData:
+      authType: credentials
+      defaultRegion: eu-west-2
```

Comment:
Do you not have to specify the `assumeRoleArn` anywhere?

Reply:
No. Grafana pod has a role specified. KIAM knows which role Grafana wants to assume.

Reply:
Apparently KIAM does that?

Reply:
Incredible. What a time to be alive.